packages/cf-component-radio/src/Radio.js

```diff
@@ -1,6 +1,86 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  cursor: theme.cursor,
+  display: theme.display,
+  minHeight: theme.minHeight,
+  paddingLeft: theme.paddingLeft,
+  marginTop: theme.marginTop,
+
+  '&:first-child': {
```

Comment:
**This works.**, However, it breaks one of the [CSS in JS principles - isolation](https://speakerdeck.com/vjeux/react-css-in-js). It's actually nicely described there. 

So what's happening? Component `Radio` is changing styles of component `Input` through a CSS first-child selector.  Component Input has no clue that this is happening. If you are an author of Input, you are losing ability to introduce safe upgrades since people "go around your API". API for our components are props. Does it make sense?

Anyway, if you don't want to thing about this now, it's ok to leave it as it is. We can revisit later.